backend/scripts/lint.sh

```
@@ -1,27 +1,26 @@
+mypy_failed=0
+pylint_failed=0
 
-run_mypy_check || mypy_failed=1  
-run_pylint_check || pylint_failed=1  
+run_mypy_check || mypy_failed=1
+run_pylint_check || pylint_failed=1
 
-# Exit with failure if any check failed  
-[ $mypy_failed -eq 0 -a $pylint_failed -eq 0 ] || exit 1  
+# Exit with failure if any check failed
+[ $mypy_failed -eq 0 -a $pylint_failed -eq 0 ] || exit 1
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)